lib/tasks/import_responsible_bodies.rake

```diff
@@ -0,0 +1,29 @@
+namespace :import do
+  desc 'Import local authorities in England'
+  task local_authorities_in_england: :environment do
+    LocalAuthoritiesInEnglandRegister.entries.each do |entry|
+      LocalAuthority
+        .where(local_authority_eng: entry['local-authority-eng'])
+        .first_or_create!(
+          name: entry['name'],
+          organisation_type: entry['local-authority-type'],
+          local_authority_official_name: entry['official-name'],
+        )
+    end
+  end
+
+  desc 'Import single and multi-academy trusts'
+  task trusts: :environment do
+    GetInformationAboutSchools.trusts_entries.each do |entry|
+      Trust
+        .where(companies_house_number: entry['Companies House Number'])
+        .first_or_create!(
+          name: entry['Group Name'],
+          organisation_type: entry['Group Type'],
+        )
+    end
+  end
```

Comment:
These aren't tested because, well ... rake task. How about moving the code here into a service to make it testable?

Reply:
Realised after I wrote that this is likely a one-off. Is that true or will this need to be re-run periodically?

Reply:
this will be a one-off, I'm pretty sure 

_(famous last words)_

Reply:
Yes, we currently don't expect to run this more than once

Reply:
@misaka I've extracted the service as suggested in 253bb9d